samples/ControlCatalog/Pages/NumericUpDownPage.xaml

```diff
@@ -76,21 +76,21 @@
       <StackPanel Orientation="Vertical" Margin="10">
         <Label Target="upDown" FontSize="14" FontWeight="Bold" VerticalAlignment="Center">Usage of decimal NumericUpDown:</Label>
         <NumericUpDown Name="upDown" Minimum="0" Maximum="10" Increment="0.5"
-                       CultureInfo="en-US" VerticalAlignment="Center" Value="{Binding DecimalValue}"
```

Comment:
Removed default culture info, as it was crashing control catalog in the browser.